R/count.R

```diff
@@ -49,19 +49,20 @@ count..data.frame <- function(.df, ..., wt = NULL, sort = FALSE, name = NULL) {
   .by <- enquos(...)
   wt <- enquo(wt)
   
-  if(quo_is_null(wt)){
-    .df <- summarize.(.df, N = .N, .by = c(!!!.by))
+  if (is.null(name)) {
+    name <- "N"
+  }
+
+  if (quo_is_null(wt)) {
+    .df <- summarize.(.df, !!name := .N, .by = c(!!!.by))
   } else {
-    .df <- summarize.(.df, N = sum(!!wt), .by = c(!!!.by))
+    .df <- summarize.(.df, !!name := sum(!!wt), .by = c(!!!.by))
   }
   
-  if(sort) {
-    .df <- .df[order(-N)]
+  if (sort) {
+    .df <- arrange.(.df, !!name)
```

Comment:
Ah good catch on needing to adjust the `sort` part, I missed that. This should now be
```r
arrange.(.df, -!!sym(name))
```
so that `name` is converted from a string to a symbol.

Reply:
I was just about to comment that `desc.()` doesn't accept `!!name` as argument. thanks

Reply:
> Error: Column 1 passed to [f]order is type 'symbol', not yet supported.
this throws the following error, we could circumvent this by :
``` r
.df[order(-get(name))]
```